twilight.py

```diff
@@ -89,21 +95,24 @@ def load_tile_matrix(self):
             if not config['DEBUG_MODE']:
                 port = serial.Serial(my_unit['serial_port'], config['SERIAL_RATE'])
                 unit_queue = queue.Queue()
-
+                rendered_frame_times = collections.deque(maxlen=FPS_FRAME_COUNT)
```

Comment:
This function is too big. Please create a TODO to split it later / split it now.